index.html

```diff
@@ -210,6 +210,22 @@ <h1 id="about-title" class="section-title">About Myself</h1>
           </section>
         </div>
       </div>
+      <section id="get-in-touch">
+        <div class="form-container">
+          <div class="contact-background">
+            <div class="contact-text">
+              <h2>Contact me</h2>
+              <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it.</p>
+            </div>
+            <form id="contact-form" action="https://formspree.io/f/meqdykzk" method="post">
+                <input type="text" name="name" maxlength="30" placeholder="Yeremias" required />
+                <input type="email" name="email" placeholder="natajayanji4@gmail.com" required />
```

Comment:
- Kindly, Use appropriate placeholder, like for name `name` or `enter your name` and some for email to make your User Interface clear. 💕👌